doc/tutorials/highlevel-bindings.md

```diff
@@ -54,6 +54,29 @@ sufficient to write (or generate) a one-to-one mirror of the C API and use that
 The generated code should still be understandable. So if writing a more idiomatic API on top of the
 direct mapping, significantly simplifies the generated code (and maybe also the template), you should write such an API.
 
+### Bindings for the `lowlevel` Library
+
+If you decide on mapping the functionality of kdb.h 1:1 it is pretty straight-forward.
+
+Your programming language of choice must provide a way to call into C code, e.g.: 
+* [cgo](https://golang.org/cmd/cgo/) for the Go programming language
+* [jni](https://docs.oracle.com/javase/8/docs/technotes/guides/jni/) for JAVA
+* [n-api](https://nodejs.org/api/n-api.html) for javascript
+
+Make sure you have a good understanding of these libraries because some of them can have unexpected downsides.
```

Comment:
About which libraries do you talk here? You mean that there might be different ways of do bindings?

In general we prefer (in this order):

- automatically and statically generated bindings (like swig)
- manually written bindings that directly call to library without any statically build part (like JNI and GI). They are a bit slower but can directly access libelektra.so.
- manually written bindings that are built statically

Reply:
Maybe library is the wrong word - I've meant the 'interop' layer to communicate with C, like cgo.

> In general we prefer (in this order):
>* automatically and statically generated bindings (like swig)
>* manually written bindings that directly call to library without any statically build part (like JNI and GI). They are a bit slower but can directly access libelektra.so.
>* manually written bindings that are built statically

Do you want me to add this?

Reply:
> Maybe library is the wrong word - I've meant the 'interop' layer to communicate with C, like cgo.

Ok.

> Do you want me to add this?

Yes, you can add it.